### PR TITLE
Change CV terms for Contact bundle

### LINKED ISSUE #1783: Change the CV term for the Contact bundle from local:contact to a real term NCIT:C47954
### Task Description

We changed the CV term for the ChadoContactTypeDefault field from local:contact to NCIT:C47954 because a real term is always better than a local made-up term.

Do the same change for the Contact bundle.

### Branch Name

tv4g0-issue1783-contact-bundle-change-cv-term

### PR DESCRIPTION
**Pull request by benjiforster**
# Tripal 4 Core Dev Task

## Description

Changed all CV terms for the Contact bundle from `local:contact` to `NCIT:C47954` and `contact` to `Communication Contact`. Also fixed a typo for `help_text` in `tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml`.

## Testing?

Functional review required.

### CODE REVIEW

**Comment by dsenalik** [comment]
Actually this will require a functional review since this changes the bundle's term which theoretically might affect something. Code changes look perfect, though. I'll try to get to this soon...

**Comment by benjiforster** [comment]
Ah, my apologies. Also - I wasn't sure if to change anything here in `tripal.tripal_content_terms.chado_content_terms.yml`:
```
-   id: 'local:contact'
name: 'contact'
description: 'An entity (e.g. individual or organization) through whom a person can gain access to information, favors, influential people, and the like.
```
I know you said not to change the id for the safe of backwards compatibility, though.

**Comment by dsenalik** [comment]
Correct, we are leaving that there unchanged on the off chance it helps people migrate Tripal 3 sites more easily, or some module uses it, but it will no longer be used anywhere in Tripal 4.

**Comment by benjiforster** [comment]
Ah, that's fine then; I believe I've changed all the CV terms.

File: tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml [hunk 1]
```diff
@@ -43,9 +43,9 @@ content_types:
         synonyms:
             - bio_data_4
 
-    -   label: Contact
-        term: local:contact
-        help_text: Use the contage page a person or institution that can be linked as a responsible party for data or results.
+    -   label: Communication Contact
```
Comment (dsenalik) [inline tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml]: I'm sorry for misleading you with incorrect suggestions, but this particular change affects the name of the actual content type, and in this case we don't want to change it. And the label is now not consistent I need to redo this suggestion but I'm not sure how 😬 - I guess I need another suggestion below...
```suggestion
    -   label: Contact
```

File: tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml [hunk 2]
```diff
@@ -43,9 +43,9 @@ content_types:
...
+        term: NCIT:C47954
+        help_text: A channel for communication between groups.
```
Comment (dsenalik) [inline tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml]: For consistency we need to revert this change, but also fix the typos.
This help text is new for Tripal 4, so the typo is not that old 😁
```suggestion
        help_text: Use the contact page for a person or institution that can be linked as a responsible party for data or results.
```

**Comment by benjiforster** [comment]
No worries. Are you able to specify exactly what to change? I can close this PR and make a new one.

**Comment by dsenalik** [comment]
No more changes needed, this is now perfect! Once tests complete, it should be ready to merge.

**Comment by benjiforster** [comment]
Great news!

**Review by dsenalik — APPROVED** [review]
I built a docker and did manual functional tests again, everything works as expected.
The test below using the devel module php form at `/devel/php` shows that the term is updated. A contact field on a project page still displays correctly.
2024-02-15_bundletermlist

**Comment by dsenalik** [comment]
Congratulation on your first merged Tripal PR 😄

**Comment by benjiforster** [comment]
Good to know, and thank you! It's also my first contribution in the world of open source 😄
